app/main.js

```diff
@@ -19,10 +19,13 @@ import { ConnectedRouter } from 'react-router-redux';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 import FontFaceObserver from 'fontfaceobserver';
 import 'normalize.css/normalize.css';
+import Detection from './utils/detection';
 import './main.scss';
-
 import configureStore from './modules/store';
 
+const detector = new Detection();
+detector.check();
```

Comment:
Z tego co pamiętam z projektów u9 to cały kod związany z wykrywaniem był w osobnym entry poincie, w którym było podbite wsparcie przez babela do starszych przeglądarek. Kwestia jest taka, że jak to jest w tym samym entry poincie webpacka i coś się wywali to ten unsupported page się nie wyświetli.